### ISSUE TITLE
warning: ‘get_granular_rand_size’ defined but not used [-Wunused-function]

### ISSUE THREAD

**Issue by ldorau** [pmem/vmemcache#174]
```
Scanning dependencies of target vmemcache_test_mt
[ 60%] Building C object tests/CMakeFiles/vmemcache_test_mt.dir/vmemcache_test_mt.c.o
In file included from /home/ldorau/work/vmemcache/tests/vmemcache_test_mt.c:43:
/home/ldorau/work/vmemcache/tests/test_helpers.h:128:1: warning: ‘get_granular_rand_size’ defined but not used [-Wunused-function]
 get_granular_rand_size(size_t val_max, size_t granularity)
 ^~~~~~~~~~~~~~~~~~~~~~
[ 62%] Linking C executable vmemcache_test_mt
[ 67%] Built target vmemcache_test_mt
Scanning dependencies of target bench_simul
[ 70%] Building C object benchmarks/CMakeFiles/bench_simul.dir/bench_simul.c.o
In file included from /home/ldorau/work/vmemcache/benchmarks/bench_simul.c:49:
/home/ldorau/work/vmemcache/tests/test_helpers.h:128:1: warning: ‘get_granular_rand_size’ defined but not used [-Wunused-function]
 get_granular_rand_size(size_t val_max, size_t granularity)
 ^~~~~~~~~~~~~~~~~~~~~~
[ 72%] Linking C executable bench_simul
[ 82%] Built target bench_simul
Scanning dependencies of target bench_micro
[ 85%] Building C object benchmarks/CMakeFiles/bench_micro.dir/bench_micro.c.o
In file included from /home/ldorau/work/vmemcache/benchmarks/bench_micro.c:45:
/home/ldorau/work/vmemcache/tests/test_helpers.h:128:1: warning: ‘get_granular_rand_size’ defined but not used [-Wunused-function]
 get_granular_rand_size(size_t val_max, size_t granularity)
 ^~~~~~~~~~~~~~~~~~~~~~
[ 87%] Linking C executable bench_micro
```

**Comment by ldorau** [comment]
Assigned @kilobyte

**Comment by ldorau** [comment]
Fixed by #171
